homeassistant/components/zwave_me/__init__.py

```diff
@@ -0,0 +1,102 @@
+"""The Z-Wave-Me WS integration."""
+import logging
+
+from zwave_me_ws import ZWaveMe, ZWaveMeData
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect, dispatcher_send
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, PLATFORMS, ZWAVE_PLATFORMS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry):
+    """Set up Z-Wave-Me from a config entry."""
+    hass.data[DOMAIN] = {}
+    hass.data[DOMAIN][entry.entry_id] = ZWaveMeController(hass, entry)
```

Comment:
Store the controller in a local variable too for convenience.
```suggestion
    controller= hass.data[DOMAIN][entry.entry_id] = ZWaveMeController(hass, entry)
```